src/widgets/fancytabwidget.cpp

```diff
@@ -383,6 +383,7 @@ void FancyTabWidget::contextMenuEvent(QContextMenuEvent *e) {
     menu_ = new QMenu(this);
     QActionGroup *group = new QActionGroup(this);
     addMenuItem(group, tr("Large sidebar"), Mode::LargeSidebar);
+    addMenuItem(group, tr("Large sidebar without description"), Mode::IconOnlySideTabs);
```

Comment:
Can we call this something simpler, perhaps: `Icons sidebar` - `Mode::IconsSidebar` ?